postman/NCG.postman_collection.json

```diff
@@ -1572,6 +1572,56 @@
 					},
 					"response": []
 				},
+				{
+					"name": "SessionRefresh",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "b514cd0a-5ec3-4814-a989-e002aa4f36ff",
+								"exec": [
+									"pm.test(\"Update central registry token\", function () {",
+									"    var jsonData = pm.response.json();",
+									"    pm.environment.set(\"CENTRAL_REGISTRY_AUTH_TOKEN\", \"bearer \" + jsonData.accessToken);",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"name": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n\t\"clientId\": \"consent-manager\",\n\t\"clientSecret\": \"{{CONSENT_MANAGER_CLIENT_SECRET}}\",\n\t\"grantType\": \"refresh_token\",\n\t\"refreshToken\": \"{{CONSENT_MANAGER_REFRESH_TOKEN}}\"\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{CENTRAL_REGISTRY_HOST}}/api/1.0/sessionsRefresh",
```

Comment:
Why can't we use the same API? and based on grant\_type decide?

Reply:
We need to pass refresh_token parameter extra for new call. The body is different for these two calls. Hence we need to use new API.

Reply:
Yeah that's one way, but typically for login, have one api, based on grant\_type pick, validate values and return response accordingly.